src/api.js

```diff
@@ -100,14 +101,37 @@ function _scoord2Geometry(scoord) {
 
 function _geometryCoordinates2scoordCoordinates(coordinates) {
   // TODO: Transform to coordinates on pyramid base layer???
-  return [coordinates[0] + 1, -coordinates[1]]
+  return [coordinates[0] + 1, -coordinates[1], 0]
```

Comment:
I think the conversion to SCOORD3D should be in dcmjs, so we don't need this third argument here.

Reply:
Done